### ISSUE TITLE
WifiDog send error

### ISSUE THREAD

**Issue by medrockstar** [wifidog/wifidog-auth-laravel#6]
Hi,

I can't connect to te server, but I have a pong :

root@OpenWrt:~# curl 'http://auth/ping?gw_id=001217DA42D2&sys_uptime=742725&sys_memfree=2604&sys_load=0.03&wifidog_uptime=3861'
Pong

But WifiDog send me error :

[7][Sat Jun 30 18:42:46 2018][6256](simple_http.c:77) Sending HTTP request to auth server: [GET /ping/?gw_id=default&sys_uptime=7791&sys_memfree=6384&sys_load=0.27&wifidog_uptime=2 HTTP/1.0
User-Agent: WiFiDog 1.2.1
Host: auth

]

[7][Sat Jun 30 18:42:46 2018][6256](simple_http.c:87) Reading response
[7][Sat Jun 30 18:42:46 2018][6256](simple_http.c:111) Read 702 bytes
[7][Sat Jun 30 18:42:46 2018][6256](simple_http.c:124) HTTP Response from Server: [HTTP/1.1 301 Moved Permanently
Date: Sat, 30 Jun 2018 18:42:46 GMT
Server: Apache/2.4.27 (Win64) PHP/5.6.31
Location: http://auth/ping?gw_id=default&sys_uptime=7791&sys_memfree=6384&sys_load=0.27&wifidog_uptime=2
Content-Length: 398
Connection: close
Content-Type: text/html; charset=iso-8859-1

<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="http://auth/ping?gw_id=default&amp;sys_uptime=7791&amp;sys_memfree=6384&amp;sys_load=0.27&amp;wifidog_uptime=2">here</a>.</p>
<hr>
<address>Apache/2.4.27 (Win64) PHP/5.6.31 Server at auth Port 80</address>
</body></html>
]
[4][Sat Jun 30 18:42:46 2018][6256](ping_thread.c:191) Auth server did NOT say Pong!
[7][Sat Jun 30 18:42:46 2018][6256](firewall.c:140) Marking auth server down

**Comment by sinkcup** [comment]
your server name is "auth" ? It's wrong. check the docs, it should be "wifidog-auth.lan"

**Comment by sithar** [comment]
Wifidog config: AuthServer {
        Hostname  wifidog.sid
        Path /
}

Im getting error
"We did not get a valid response from the central server"

Debug report:

Server: Apache/2.4.33 (Unix) OpenSSL/1.0.2n PHP/7.2.4 mod_perl/2.0.8-dev Perl/v5.16.3
Location: http://wifidog.sid/auth?stage=login&ip=192.168.101.187&mac=08:00:27:3c:7b:01&token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOjEsImlzcyI6Imh0dHA6Ly93aWZpZG9nLnNpZC9sb2dpbiIsImlhdCI6MTUzNDE0MzU2NSwiZXhwIjoxNTM0MTQ3MTY1LCJuYmYiOjE1MzQxNDM1NjUsImp0aSI6IjJzOGRjT2ltUm9xWHhlUU0ifQ.jp9quwj-sg4P4b35rsEMN3V4HfYKr4JZoCSFgqVYApg&incoming=0&outgoing=0&gw_id=08002773A433
Content-Length: 595
Connection: close
Content-Type: text/html; charset=iso-8859-1

<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="http://wifidog.sid/auth?stage=login&amp;ip=192.168.101.187&amp;mac=08:00:27:3c:7b:01&amp;token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOjEsImlzcyI6Imh0dHA6Ly93aWZpZG9nLnNpZC9sb2dpbiIsImlhdCI6MTUzNDE0MzU2NSwiZXhwIjoxNTM0MTQ3MTY1LCJuYmYiOjE1MzQxNDM1NjUsImp0aSI6IjJzOGRjT2ltUm9xWHhlUU0ifQ.jp9quwj-sg4P4b35rsEMN3V4HfYKr4JZoCSFgqVYApg&amp;incoming=0&amp;outgoing=0&amp;gw_id=08002773A433">here</a>.</p>
</body></html>
]
[7][Mon Aug 13 12:59:25 2018][3768](auth.c:159) Locking client list
[7][Mon Aug 13 12:59:25 2018][3768](auth.c:159) Client list locked
[3][Mon Aug 13 12:59:25 2018][3768](auth.c:192) Got ERROR from central server authenticating token eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOjEsImlzcyI6Imh0dHA6Ly93aWZpZG9nLnNpZC9sb2dpbiIsImlhdCI6MTUzNDE0MzU2NSwiZXhwIjoxNTM0MTQ3MTY1LCJuYmYiOjE1MzQxNDM1NjUsImp0aSI6IjJzOGRjT2ltUm9xWHhlUU0ifQ.jp9quwj-sg4P4b35rsEMN3V4HfYKr4JZoCSFgqVYApg from 



Where am i going wrong? 
Please help

**Comment by sinkcup** [comment]
@sithar do you have config TrustedMACList?

**Comment by sithar** [comment]
TrustedMACList 08:00:27:0A:6C:7B,AC:BC:32:D0:D7:71,08:00:27:73:A4:33,08:00:27:13:A7:6B,08:00:27:2E:25:4A


listing all network interfaces of auth server and openwrt .

**Comment by sinkcup** [comment]
follow the standard, use .lan domain.
open laravel debug log in `storage/logs`, then curl auth server on the router, like:

```
curl http://wifidog.lan/auth?stage=login&ip=192.168.101.187&mac=08:00:27:3c:7b:01&token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOjEsImlzcyI6Imh0dHA6Ly93aWZpZG9nLnNpZC9sb2dpbiIsImlhdCI6MTUzNDE0MzU2NSwiZXhwIjoxNTM0MTQ3MTY1LCJuYmYiOjE1MzQxNDM1NjUsImp0aSI6IjJzOGRjT2ltUm9xWHhlUU0ifQ.jp9quwj-sg4P4b35rsEMN3V4HfYKr4JZoCSFgqVYApg&incoming=0&outgoing=0&gw_id=08002773A433
```

your web server IP is 192.168.1.10, but client device is 192.168.101.187?
try to open the web server homepage on the client device's brower.

**Comment by henok400** [comment]
I got similar error No valid answer from the central server 
I must use HTTPS for my wifidog config
```
AuthServer {
Hostname wan.cloudwifizone.com
HTTPPort 80
SSLAvailable yes
Path /
}
```
Someone can you fix this Im using openwrt v19.07 wifidog v1.3.1-4
